packages/fxa-auth-client/test/client.ts

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as assert from 'assert';
```

Comment:
Not sure if we want to use [strict assertion mode](https://nodejs.org/docs/latest-v20.x/api/assert.html#strict-assertion-mode).
> _In strict assertion mode, non-strict methods behave like their corresponding strict methods. For example, [`assert.deepEqual()`](https://nodejs.org/docs/latest-v20.x/api/assert.html#assertdeepequalactual-expected-message) will behave like [`assert.deepStrictEqual()`](https://nodejs.org/docs/latest-v20.x/api/assert.html#assertdeepstrictequalactual-expected-message)._
>
> _In strict assertion mode, error messages for objects display a diff. In legacy assertion mode, error messages for objects display the objects, often truncated._

Reply:
I didn't know about this mode. Which do you think is better? 

Other tests in this suite aren't using strict, and the for the test here it wouldn't matter, fwiw.

Reply:
I usually lean towards .strict mode unless there's a reason not to, but like you say, we don't seem to use it elsewhere in the repo.

```sh
git grep -Pho "\bassert.equal\(" -- apps libs packages | sort -i | uniq -c | sort -nr
7972 assert.equal(

git grep -Pho "\bassert[\./]strict(\w+)?" -- apps libs packages | sort -i | uniq -c | sort -nr
 414 assert.strictEqual
```